docs_source_files/content/support_packages/mouse_and_keyboard_actions_in_detail.de.md

```diff
@@ -57,7 +57,34 @@ searchBtn = driver.find_element_by_link_text("Sign in")
 webdriver.ActionChains(driver).click_and_hold(searchBtn).perform()
   {{< / code-panel >}}
   {{< code-panel language="csharp" >}}
-// Please raise a PR
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Interactions;
+
+namespace SeleniumApp
+{
+    public class ClickAndHold
+    {
+        public static void Main(string[] agrs)
+        {
+            IWebDriver driver = new ChromeDriver();
+            try
+            {
+                // Navigate to Url
+                driver.Navigate().GoToUrl("https://google.com");
+                // Store 'google search' button web element
+                IWebElement searchBtn = driver.FindElement(By.LinkText("Sign in"));
+                Actions actionProvider = new Actions(driver);
+                // Perform click-and-hold action on the element
+                actionProvider.ClickAndHold(searchBtn).Build().Perform();
+            }
```

Comment:
Hi @hiroksarker,

Code looks way too indented to the right, can you please align it?